documentation/src/docs/asciidoc/release-notes-5.0.0-M4.adoc

```diff
@@ -34,6 +34,8 @@ on GitHub.
 * `--hide-details` option of the `ConsoleLauncher` is deprecated, use `--details none` instead.
 * The ZIP distribution containing the console launcher is no longer provided. It is replaced
   by an executable standalone JAR distribution. For details, see "New Features" section below.
+* ExtensionContext composite Namespaces with the same parts but in different order will
+  not be considered equal.
```

Comment:
Thanks for you comment @jbduncan. I couldn't find something that sounded good to me. Your suggestion hits the mark :).

Going back to change this, I also found I added this at the wrong section :(, so fixed that new with my new commit.